src/main/java/gov/cms/madie/cqllibraryservice/controllers/CqlLibraryController.java

```diff
@@ -120,6 +123,29 @@ public ResponseEntity<CqlLibrary> updateCqlLibrary(
         .orElseThrow(() -> new ResourceNotFoundException("CQL Library", id));
   }
 
+  @GetMapping(value = "/cql", produces = MediaType.TEXT_PLAIN_VALUE)
+  public String getLibraryCql(
+      @RequestParam String name,
+      @RequestParam String version,
+      @RequestParam Optional<String> model) {
+    List<CqlLibrary> libs =
+        model.isPresent()
+            ? cqlLibraryRepository.findAllByCqlLibraryNameAndDraftAndVersionAndModel(
+                name, false, Version.parse(version), model.get())
+            : cqlLibraryRepository.findAllByCqlLibraryNameAndDraftAndVersion(
+                name, false, Version.parse(version));
+    if (libs == null || libs.isEmpty()) {
```

Comment:
```suggestion
    if (CollectionUtils.isEmpty(libs)) {
```
Does the same thing 🤷‍♂️ Take your pick.